db/migrate/20210824160127_update_notified_at_with_created_at.rb

```diff
@@ -0,0 +1,13 @@
+class UpdateNotifiedAtWithCreatedAt < ActiveRecord::Migration[6.1]
+  def change
+    reversible do |dir|
+      dir.up do
+        PlanningApplication.all.each do |planning_application|
```

Comment:
still `find_each`

Reply:
I tried find_each on the console and this:

```
 PlanningApplication.find_each do |planning_application| planning_application.validation_requests.each do |request| request.created_at end end
 ```
 returns: 
![image](https://user-images.githubusercontent.com/9452321/130660187-d6cef973-3c36-4b0c-a475-8757b2aff1a9.png)

whereas this:

``` PlanningApplication.all.each do |planning_application| planning_application.validation_requests.each do |request| request.created_at end end```
returns

![image](https://user-images.githubusercontent.com/9452321/130660354-afe4f7db-6934-4b76-9148-2ad54dce2528.png)

so I don't get it, find_each doesn't return the data?

Reply:
it doesn't; it just yields each record, but you don't need the returned data do you?